index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1>Hello microverse</h1>
```

Comment:
- [ ] Please add a readme file to your branch using the microverse readme [template](https://github.com/microverseinc/readme-template)🙏. When adding the file make sure to personalize the template by filling out all sections (only the FAQ section is optional). The readme file is very important as it provides vital documentation on what your project is about and how to get it up and running👍.

You can use[ this repo](https://github.com/KenbonTN/Hello-Microverse-1/tree/feature-branch) as a guide when editing the README template.